tests/integration/test_integration.py

```diff
@@ -1925,31 +1913,41 @@ def test_ks_2sample_constraint_perfect_between(engine, int_table1, data):
     assert operation(test_result.outcome), test_result.failure_message
 
 
-# TODO: Enable this test once the bug is fixed.
-@pytest.mark.skip(reason="This is a known bug and unintended behaviour.")
```

Comment:
No longer skipping this test (as well as adding further examples) should indicate that this PR solves https://github.com/Quantco/datajudge/issues/42